br-webui/views/network.liquid

```diff
@@ -296,5 +370,23 @@ socket.on('current ip', function(data) {
 		</div>
 	</div>
 </div>
+<script>
 
+function onToggleAdvancedSwitch() {
+	var div = document.getElementById("eth-conf");
+	console.log(div);
```

Comment:
I believe only the top level element needs to be modified.

Reply:
try this 	var div = document.getElementById("eth-conf");
	console.log(div);
	console.log("Entered function");


		if (document.getElementById("advanced-checkbox").checked) {
        	div.style.display = "block";
		} else {
	   	 	div.style.display = "none";
		}

Reply:
This is working. I already pushed a commit for that. but in the beginning the options show up. only once you check and uncheck the toggle switch do they disappear. for fixing that, i was thinking of doing this:
function hideAdvancedOptions() {
	var div = document.getElementsByClassName("advanced");
	for (var i=0; i<div.length; i++) {
		div[i].style.display = "none";
	}
}

function onToggleAdvancedSwitch() {
	var div = document.getElementsByClassName("advanced");
	console.log(div);
	for (var i=0; i<div.length; i++) {
		console.log(div[i]);
		if (document.getElementById("advanced-checkbox").checked) {
        	div[i].style.display = "none";
		} else {
	   	 	div[i].style.display = "block";
		}
	}
}

hideAdvancedOptions();

Reply:
i also pushed another commit for this. you can try it out. let me know if thats not how you want to do it.